### ISSUE TITLE
Unify casing in Chart APIs

### ISSUE THREAD

**Issue by kMutagene** [plotly/Plotly.NET#105]
### Description

Basically all Chart extension functions should be lower camel case (`Chart.someFunctionName`). The API is kind of inconsistent in that regard:

see e.g.`Chart.Show` or `Chart.Combine` vs. `Chart.withTitle` or `Chart.withX_AxisStyle`

This is a change that is absolutely not backwards compatible but i think it should be done before finally releasing 2.0.0

However, i would keep the core Chart API the same to make the those look more like constructors (meaning we keep `Chart.Point`, `Chart.Line`, etc.)

**Comment by WhiteBlackGoose** [comment]
~~As per discord discussion: types like `FontFamily` , `Fill` and where needed to be named consistent with F# (with no underscores)~~
there will be a new issue for this one
